subsys/fs/fuse_fs_access.c

```diff
@@ -15,6 +18,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/mount.h>
+#include <sys/stat.h>
```

Comment:
Is this really needed?

Reply:
It's not needed to fix the compilation error, indeed, so I don't know whether it's good practice or not to add this include? I think that's where the macros are actually declared?

Reply:
I'd consider it more correct to include it yes.